Library/Formula/percona-server.rb

```diff
@@ -86,6 +86,10 @@ def install
       -DWITHOUT_DIALOG=1
     ]
 
+    # TokuDB is broken on MacOsX
+    # https://bugs.launchpad.net/percona-server/+bug/1531446
+    args.concat %W[-DWITH_TOKUDB=1]
```

Comment:
If TokuDB is broken do we not want to disable it by default rather than enable it? I have no idea about Percona so please feel to disagree!